ansible/roles/kernel/tasks/os/redhat-8.yml

```diff
@@ -23,6 +23,7 @@
     - kernel-tools
     - kernel-tools-libs
     #- kernel-tools-libs-devel
+    - '{{ ansible_distribution | lower }}-release'
```

Comment:
What is this line doing specifically?

Reply:
Upgrading the `rocky-release` package noted in the PR description. This ensures correct detection of the OS version upon reboot.

Reply:
It is written anticipating the possibility of supporting AlmaLinux (where the package name is different but follows the pattern)